libraries/points/src/index.ts

```diff
@@ -13,3 +13,4 @@ export {
 export { SetReader, type SetReaderParams } from './set-reader.js'
 export { SetWriter, type SetWriterFromSeedParams, type SetWriterParams } from './set-writer.js'
 export type { QueryDocumentsOptions, QueryDocumentsResult } from './types.js'
+export { getAuthenticatedCeramic, getCeramic } from './ceramic.js'
```

Comment:
Not a big deal if you really need them, but I think these should be kept as internal to the library rather than a public API.
If it's useful beyond this library, maybe they can be extracted to a dedicated library?

Reply:
I think it's useful - it's necessary for my server to access the client (if you look back at the prior versions of /utils/context.ts, I was previously importing from did-utils, but that wasn't pushed up into a lib)

Reply:
Yes so if we need a lib like did-utils to be public I think that would be a better option, so at least it's a specific-purposed lib and API rather than having to use the internals of the points library, no?